src/auditor/checks.py

```diff
@@ -61,6 +62,35 @@ def get_group_from_table(metatable_dict_entry):
     return group
 
 
+def get_item_properties(item):
+    """
+    Gets all the relevant info about item in successive calls; seems to speed up times by about .5 to 1 second per item.
+    """
+    ItemProperties = namedtuple(
+        'ItemProperties',
+        ['title', 'tags', 'shared_with', 'itemid', 'protected', 'description', 'metadata', 'content_status', 'layers']
+    )
+
+    title = item.title
+    tags = item.tags
+    try:
+        shared_with = item.shared_with
+    except:  # pylint: disable=bare-except
+        shared_with = Exception
```

Comment:
```suggestion
    except Exception as ex:
        shared_with = ex
```
**question**: Why not something like this? Or is there a reason why you want the Exception class for later on?